core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb.go

```diff
@@ -228,6 +215,10 @@ func newVersionedDB(couchInstance *couchInstance, redoLogger *redoLogger, dbName
 		return nil, err
 	}
 
+	if err = vdb.initChannelMetadata(savepoint == nil, nsProvider); err != nil {
```

Comment:
To improve the readability, we can have

```
isNewDB := savepoint == nil
if err = vdb.initChannelMetadata(isNewDB, nsProvider)
```

This helps in providing a context for `savepoint == nil`.